pallets/payment-streams/src/lib.rs

```diff
@@ -215,6 +215,10 @@ pub mod pallet {
     #[pallet::storage]
     pub type AccumulatedPriceIndex<T: Config> = StorageValue<_, BalanceOf<T>, ValueQuery>;
 
+    // List of priviledge providers
```

Comment:
Using `///` for rust docs makes it easier then to hover over the type where it's used, and see what it is about.

```suggestion
    /// Mapping of Privileged Providers.
    ///
    /// Privileged Providers are those who are allowed to charge up to the current tick in
    /// fixed rate payment streams, regardless of their [`LastChargeableInfo`].
```